src/ensembl/src/content/app/browser/track-panel/track-panel-list/TrackPanelListItem.tsx

```diff
@@ -228,7 +228,9 @@ export const TrackPanelListItem = (props: TrackPanelListItemProps) => {
 
     if (allTrackLabelsOn && isTurnedOn) {
       genomeBrowser?.send({
-        type: OutgoingActionType.TURN_ON_LABELS,
+        type: allTrackLabelsOn
```

Comment:
Would `enable` be better here and below than `on`?

Reply:
I think both are fine unless one makes it hard to understand.